simtools/applications/print_array_elements.py

```diff
@@ -96,10 +98,16 @@ def _parse(label=None, description=None):
     config = configurator.Configurator(label=label, description=description)
 
     config.parser.add_argument(
-        "--array_element_list",
-        help="list of array element positions (ecsv format)",
+        "--input",
```

Comment:
Apologies for jumping in the middle of the review, but these options `input/input_meta` are expected to be general to many applications, no? Why not add them to the configurator instead?

Reply:
I am aware of this good point, however I would like to consciously contain these changes first to print_array_elements and only after we are happy with this generalize it to most/all applications. Moving it to the configurator might or might break other applications now, I don't want to fix these interdependencies with this PR.

After this PR, I first want to add the input validation to print_array_elements. Then we look at it together, possible change something and only then propagate it everywhere.

Reply:
OK!